plugins/org.jkiss.dbeaver.model.rcp/src/org/jkiss/dbeaver/model/rcp/DesktopWorkspaceImpl.java

```diff
@@ -318,14 +318,17 @@ public void deleteProject(@NotNull DBPProject project, boolean deleteContents) t
         if (!(project instanceof RCPProject rcpProject)) {
             throw new DBException("Project '" + project.getName() + "' is not an RCP project");
         }
+        if (project == activeProject) {
```

Comment:
But user will still see the Delete button in the context menu, as far as I understand.

Reply:
There is another verification for that